### PR TITLE
[CLOUD-3168] Missing escape in context.xml sed in os-jws-secure-mgmt-console module

### PR DESCRIPTION
**Pull request by csutherl**
Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by csutherl** [comment]
I didn't create a CLOUD jira first (I didn't know that was still a thing), so I'll do that and update the title/description, etc later so this can be accepted.

**Comment by csutherl** [comment]
I re-pushed with my signature. Will do it again to add the CLOUD issue.

**Comment by csutherl** [comment]
https://issues.jboss.org/browse/CLOUD-3168

File: os-jws-secure-mgmt-console/run [hunk 1]
```diff
@@ -7,4 +7,4 @@ ADDED_DIR=${SCRIPT_DIR}/added
 
 cp -p "$ADDED_DIR/secure-mgmt-console.sh" "$JWS_HOME/bin/launch/"
 
-sed -i -e 's#</Context>#\n   <Valve className="org.apache.catalina.valves.RemoteAddrValve" allow="127\.\d+\.\d+\.\d+|::1|0:0:0:0:0:0:0:1"/>\n</Context>#' $JWS_HOME/webapps/manager/META-INF/context.xml
+sed -i -e 's#</Context>#\n   <Valve className="org.apache.catalina.valves.RemoteAddrValve" allow="127\\.\\d+\\.\\d+\\.\\d+|::1|0:0:0:0:0:0:0:1"/>\n</Context>#' $JWS_HOME/webapps/manager/META-INF/context.xml
```
Comment (jmtd) [inline os-jws-secure-mgmt-console/run]: LGTM

**Pull request merged**